### PR TITLE
docs: Update installing extensions page

### LINKED ISSUE #6883: Update documentation on how to install extensions
### Is your enhancement related to a problem? Please describe

The current documentation does not reflect the details on how to install extensios 

### Describe the solution you'd like

Update existing documentation to include details on how to install extensions

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cdrage**
docs: Update installing extensions page

### What does this PR do?

Updates the page on how to install extensions into something more
graphical / up to date.

### Screenshot / video of UI

See website link / documentation markdown.

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/6883

### How to test this PR?

View docs

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: website/docs/extensions/install/index.md [hunk 1]
```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
...
 
-- The extension Open Container Initiative (OCI) image name, such as: `quay.io/fbenoit/my-first-extension`.
-
-  See [Publishing a Podman Desktop extension](/docs/extensions/publish).
+- Extensions are packaged as container images. The extension will either direct you to the catalog tab, or provide you with a link to the image (ex. `quay.io/example/extension-name`).
 
 #### Procedure
 
-1. Go to the **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Extensions**.
+1. Click on the extensions icon.
```
Comment (deboer-tim) [inline website/docs/extensions/install/index.md]: '**Extensions** icon'. Since it is a UI element like the Containers page it should be capitalized.

File: website/docs/extensions/install/index.md [hunk 2]
```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
...
+
+![](../img/extensions-icon.png)
+
+2. Browse the Catalog and install the extension.
```
Comment (deboer-tim) [inline website/docs/extensions/install/index.md]: `Catalog` is a UI element, should be bold.

File: website/docs/extensions/install/index.md [hunk 3]
```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
...
+
+![](../img/browse-catalog.png)
 
-1. In the **Name of the Image** field, write the extension OCI image name.
+3. Alternatively, you can also press "Install custom..." and provide the image link.
```
Comment (deboer-tim) [inline website/docs/extensions/install/index.md]: Likewise I think think it should just be **Install custom...** in bold, no quotes to be consistent.

File: website/docs/extensions/install/index.md [hunk 4]
```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
...
 
-1. Click the **Install extension from the OCI image** button.
+![](../img/install-custom.png)
 
 #### Verification
 
-- If Podman Desktop satisfies the minimum required version of the extension, the extension activates.
+- The extension can be verified by checking the "Installed" section of the Extensions page.
```
Comment (deboer-tim) [inline website/docs/extensions/install/index.md]: **Installed** bold.

File: website/docs/extensions/install/index.md [hunk 5]
```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
...
 
-Consider installing Podman Desktop extensions to enrich the default capabilities of Podman Desktop.
+Installing an extension is a great way to expand the capability of Podman Desktop.
 
 #### Prerequisites
 
-- The extension Open Container Initiative (OCI) image name, such as: `quay.io/fbenoit/my-first-extension`.
-
-  See [Publishing a Podman Desktop extension](/docs/extensions/publish).
+- Extensions are packaged as container images. The extension will either direct you to the catalog tab, or provide you with a link to the image (ex. `quay.io/example/extension-name`).
```
Comment (deboer-tim) [inline website/docs/extensions/install/index.md]: 'Extensions are packaged as container images.' feels irrelevant/too much detail here. Maybe could be in the custom section, like 'extensions can be installed from their container image'.

'The extension will ...' - it feels like it should be worded by starting with "There are two ways to install an extension, ..."

**Comment by cdrage** [comment]
Updated, I also added a small note to WEBSITE_CONTRIBUTING within this PR with that tip about bolding references to UI elements.

I feel as though honestly we dont need the prerequisites part? Like.. the prerequisite being "you have the link of the extension" seems a bit odd, so I've removed it to keep it simplified. Let me know what you think.

**Review by deboer-tim — APPROVED** [review]
